src/components/DocumentVideo/reusables/style.scss

```diff
@@ -0,0 +1,91 @@
+@import '../../Theme/constants';
+
+.progress {
+  align-self: center;
+  background-color: rgba(255, 255, 255, 0.4);
+  border-radius: 4 * $unit;
+  color: $color-white;
+  font-size: var(--font-size-small);
+  margin-top: 16 * $unit;
+  padding: 4 * $unit 10 * $unit;
+  text-transform: uppercase;
+}
+
+.instructions {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.instructions > span {
+  color: $color-white;
+  font-family: var(--osdk-font-family-body);
+  font-size: var(--font-size-base);
+  margin-bottom: 8 * $unit;
+
+  @media (--medium-viewport) {
+    font-size: var(--font-size-x-small);
+    margin-bottom: 0;
+  }
+}
+
+.instructions > .title {
+  font-size: 24 * $unit;
+  font-size: 125%;
+  font-weight: 600;
+
+  @media (--medium-viewport) {
+    font-size: var(--font-size-large);
+  }
+}
+
+.instructions .success {
+  background-image: url('./assets/success.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 48 * $unit;
+}
+
+.instructions .ariaLabel {
+  opacity: 0;
```

Comment:
I've tested on iPhone and Samsung Galaxy Note, and Charlotte tested on Google Pixel and it worked. The point here is when I hid the elements using `display: none`, they weren't spoken at all. So I put this as a simple workaround. But I totally agree that this isn't the correct way to do that. Just didn't have enough time to investigate the complete solution. IMO this could be kept as-is for now and we can always tackle it later in an accessibility initiative.